subtests/docker_cli/systemd_in_container/systemd_in_container.py

```diff
@@ -0,0 +1,50 @@
+r"""
+Summary
+---------
+
+Verify running systemd as ENTRYPOINT in container.
+
+Operational Summary
+----------------------
+
+#. Execute bundled test.sh script to perform label checking operations.
+#. Fail subtest if script returns non-zero.
+
+Prerequisites
+---------------
+
+Commands contained w/in test script are available/functional on system.
+Specifically docker 1.12 is needed for dockerd and containerd.
+"""
+
+from os.path import join
+from autotest.client.utils import run
+from dockertest import subtest
+from dockertest.images import DockerImages
+from dockertest.output.validate import mustpass
+from dockertest.output.dockerversion import DockerVersion
+
+
+class systemd_in_container(subtest.Subtest):
+
+    def initialize(self):
+        # See Prerequisites (above)
+        DockerVersion().require_server("1.12")
+        self.stuff['result'] = None
+        self.stuff['di'] = DockerImages(self)
```

Comment:
Frankly, this is just a copy of subtests/docker_cli/selinux_labels/selinux_labels.py without me giving much thoughts to how things are being done. So whatever is the best practice these days, let's do it.